src/renderer/components/Player.tsx

```diff
@@ -32,14 +33,16 @@ function filterPathsToJustImages(imageTypeFilter: string, paths: Array<string>):
   return paths;
 }
 
-const keyMap2 = {
-  playPause: 'space',
-  historyBack: 'left',
-  historyForward: 'right',
-  navigateBack: 'backspace',
-  toggleFullscreen: 'CommandOrControl+F',
+const keyMap = {
+  playPause: ['Play/Pause', 'space'],
+  historyBack: ['Back in time', 'left'],
+  historyForward: ['Forward in time', 'right'],
+  navigateBack: ['Go back to scene details', 'backspace'],
+  toggleFullscreen: ['Toggle fullscreen', 'CommandOrControl+F'],
 };
 
+let originalMenu = Menu.getApplicationMenu();
```

Comment:
It's a crappy hack to keep this in a local variable, but bringing it in the class means writing a custom constructor, which sucks.